sale_order_line_date/__manifest__.py

```diff
@@ -8,7 +8,7 @@
 
 {
     "name": "Sale Order Line Date",
-    "version": "15.0.1.0.1",
+    "version": "15.0.1.0.2",
```

Comment:
You don't need to update this unless you don't have migration scripts. It will be done by the merger with ocabot

Reply:
Ok, I'll change it then